### ISSUE TITLE
Recaptcha is not fully accessible

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/dimes#881]
## Describe the bug
In an internal accessibility audit, multiple issues were identified with our current react-google-recaptcha implementation with screen reader and keyboard testing. These include insufficient color contrast in focus styles, lack of keyboard focus on the ties of the recaptcha visual test, and the recaptcha visual test being cutoff on mobile/zoom (at 320px).

Additionally, the input error message is not currently programmatically associated with the input (although we have implemented a workaround so that the error message itself gets keyboard focus when there is an error).

## To reproduce
See internal audit issues 134-136.

## Expected behavior
All recaptcha functionality should be accessible on all screen sizes, with zoom/magnification, and with a keyboard.

## Additional context
We updated to react-google-recaptcha 3 in our [update to React 19](https://github.com/RockefellerArchiveCenter/dimes/pull/829). Double check this package for additional updates and/or implementation guidance related to accessibility.
